nodejs/week1/avg.js

```diff
@@ -0,0 +1,10 @@
+let total = 0;
+for (let i = 2; i < process.argv.length; i++) {
+  total += +process.argv[i];
```

Comment:
If we add an if condition just above this, to check if the value is Number or not. It will increase the performance of the program, so, type checking is done before calculating the average. :)